lib/workers/pr/changelog/release-notes.ts

```diff
@@ -2,64 +2,58 @@ import * as URL from 'url';
 import is from '@sindresorhus/is';
 import { DateTime } from 'luxon';
 import MarkdownIt from 'markdown-it';
-import { PLATFORM_TYPE_GITLAB } from '../../../constants/platforms';
 import { logger } from '../../../logger';
 import * as memCache from '../../../util/cache/memory';
 import * as packageCache from '../../../util/cache/package';
-import * as hostRules from '../../../util/host-rules';
 import { linkify } from '../../../util/markdown';
 import * as github from './github';
 import * as gitlab from './gitlab';
-import type { ChangeLogFile, ChangeLogNotes, ChangeLogResult } from './types';
+import type {
+  ChangeLogFile,
+  ChangeLogNotes,
+  ChangeLogProject,
+  ChangeLogResult,
+} from './types';
 
 const markdown = new MarkdownIt('zero');
 markdown.enable(['heading', 'lheading']);
 
 export async function getReleaseList(
-  apiBaseUrl: string,
-  repository: string
+  project: ChangeLogProject
 ): Promise<ChangeLogNotes[]> {
   logger.trace('getReleaseList()');
-  // istanbul ignore if
-  if (!apiBaseUrl) {
-    logger.debug('No apiBaseUrl');
-    return [];
-  }
+  const { apiBaseUrl, repository, type } = project;
   try {
-    if (apiBaseUrl.includes('gitlab')) {
+    if (type === 'gitlab') {
       return await gitlab.getReleaseList(apiBaseUrl, repository);
     }
 
-    const opts = hostRules.find({
-      hostType: PLATFORM_TYPE_GITLAB,
-      url: apiBaseUrl,
-    });
-    if (opts.token) {
-      return await gitlab.getReleaseList(apiBaseUrl, repository);
+    if (type === 'github') {
+      return await github.getReleaseList(apiBaseUrl, repository);
     }
-
-    return await github.getReleaseList(apiBaseUrl, repository);
   } catch (err) /* istanbul ignore next */ {
     if (err.statusCode === 404) {
-      logger.debug({ repository }, 'getReleaseList 404');
+      logger.debug({ repository, type, apiBaseUrl }, 'getReleaseList 404');
     } else {
-      logger.info({ repository, err }, 'getReleaseList error');
+      logger.debug(
+        { repository, type, apiBaseUrl, err },
+        'getReleaseList error'
+      );
     }
-    return [];
   }
+  return [];
```

Comment:
Should we log here that no supported type was found?

Reply:
done